jdi-light-mobile-examples/src/main/java/io/github/com/sections/LoginForm.java

```diff
@@ -0,0 +1,18 @@
+package io.github.com.sections;
+
+import com.epam.jdi.light.elements.composite.Form;
+import com.epam.jdi.light.elements.pageobjects.annotations.Root;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.XPath;
+import io.github.epam.entities.User;
+import org.openqa.selenium.WebElement;
+
+public class LoginForm extends Form<User> {
+	@UI("*root*//header//form//input[@type='text']")
+	public WebElement name;
+	@Css("input[type=password]") WebElement password;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5104364)